specs/APUs-MOBILE/Phoenix/Ryzen-7-7840HS.yaml

```diff
@@ -0,0 +1,17 @@
+name: Ryzen-7-7840HS
+humanName: Ryzen 7 7840HS
+isPart: true
+type: CPU
+inherits:
+- Phoenix
+data:
+  L1 Cache (Data): 256 KiB
+  L1 Cache (Instruction): 256 KiB
+  L2 Cache (Total): 8 MiB
+  L3 Cache (Total): 16 MiB
+  TDP: 35-54 W
```

Comment:
We should probably add a new field indicating cTDP support

Reply:
Would you rather it be like
```TDP:
cTDP up:
cTDP down:```

Reply:
That's reasonable, I also think it's reasonable to do `TDP: 35-54 W` plus `cTDP support: true`, choose whatever way you think is clearer.